modules/serverless-multi-container-support-for-serving.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-configuring-multi-container-service_{context}"]
+= Configuring a multi-container service
+
+.Procedure
+
+. To enable Knative Serving to support multiple containers, modify the `KnativeServing` custom resource (CR) to include the following YAML:
```

Comment:
This is [enabled by default](https://github.com/openshift-knative/serving/blob/release-v1.8/config/core/configmaps/features.yaml#L50-L54), same as upstream:
```
    # Indicates whether multi container support is enabled
    #
    # WARNING: Cannot safely be disabled once enabled.
    # See: https://knative.dev/docs/serving/feature-flags/#multi-containers
    multi-container: "enabled"
```